genotype/store/cli.py

```diff
@@ -105,6 +109,28 @@ def sample(context, sample_id):
         context.abort()
 
 
+@click.command('prepare-trending')
+@click.option('-s', '--sample-id', 
+                help = 'return sample with specific sample id.')
+@click.option('-d', '--days',
+                help = 'return samples added a specific number of days ago.')
+@click.pass_context
+def prepare_trending(context, days, sample_id):
+    """Get a sample/samples from the database in mongo doc format."""
```

Comment:
The "presentation" should be made in the app that needs the data in a specific manner

Reply:
Not sure what you mean by presentation. The doc string?

Reply:
preparing the data for the end use should probably be done at the  end application, seems like we are preparing trending already at this stage?

Reply:
or at least in cg...

Reply:
could you please move the prepare_trending into cg or motivate why should stay here?

Reply:
Alright guys. The point is that as the code stands now we have strict coupling between genotype and vogue. Which is a bad thing (tm). There is a lot more explanation in Clean Code for Python (p 70-71 and Chapter 10). The rule of thumb is that you want to have low coupling and high cohesion;
- to minimize ripple effects: changing code in one place warrants a change in the coupled side
- low level of abstraction: it is hard to see this function solving any other problem than for vogue
- no code reuse: if code is written to be consumed by another function and only for that function, there is little chance we will reuse this code somewhere else.

In any case: what to do?

- Why don't we make a general export function here? How it needs to be prepared, formatted, etc... for the program consuming the export is something that cg needs to handle. That will solve all of the above mentioned problems.
- Don't _name_ anything for the coupled side. There is no reason to name any of the functions here 'trending' except that it will be used be vogue. Meaning: there is low level of abstraction.